### PR TITLE
Save network manager config

### PR DESCRIPTION
**Pull request by imobachgs**
Write NetworkManager configuration at the end of the installation process.

* Live installer: just copy the files under `/etc/NetworkManager/system-connections` as it is already using NetworkManager.
* Normal installation: use the new support for NetworkManager to write the configuration.

Manually tested the regular and the live installer.

Trello: https://trello.com/c/EdjoT1NN/

### CODE REVIEW

File: src/lib/network/clients/save_network.rb [hunk 1]
```diff
@@ -298,12 +291,32 @@ def configure_lan
...
 
+    # Configures NetworkManager
+    #
+    # When running the live installation, it is just a matter of copying
+    # system-connections to the installed system. In a regular installation,
+    # write the settings in the Yast::Lan.yast_config object.
+    def configure_network_manager
+      return unless Y2Network::ProposalSettings.instance.network_service == :network_manager
+
+      if Yast::Mode.live_installation
```
Comment (teclator) [inline src/lib/network/clients/save_network.rb:312]: This mode is still valid? I think that we were not able to rely on it anymore and it is detected as normal mode or something like that.

Comment (teclator) [reply src/lib/network/clients/save_network.rb:312]: We probably should document it in the code:

See https://trello.com/c/ueqrCN8I/3630-improve-live-installation-usability-and-behavior

**Review by teclator — APPROVED** [review]
LGTM

File: src/lib/network/clients/save_network.rb (hunk 1 above)
Comment (imobachgs) [reply src/lib/network/clients/save_network.rb]: Good point. I have updated the code to rely on `Yast::Lan.system_config.backend` as you suggested offline.

**Pull request merged**